events/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Use the official Python image.
+# https://hub.docker.com/_/python
+FROM python:3.7
+
+# Copy local code to the container image.
+ENV APP_HOME /app
+WORKDIR $APP_HOME
+COPY . .
+
+# Install production dependencies.
+RUN pip install Flask gunicorn pymongo[srv]
```

Comment:
Gabe just switched the other path to use a `requirements.txt` in #2 

You might want to do that as well.